iOS/MSMapsModules/MSMapsModules/MSGeoJsonParser/MSGeoJsonParser.m

```diff
@@ -0,0 +1,138 @@
+//
+//  MSGeoJsonParser.m
+//  MapsModulesSample
+//
+//  Created by Elizabeth Bartusiak (t-elbart) on 6/12/20.
+//  Copyright © 2020 Microsoft.
```

Comment:
@KristofferHenriksson Do you know if we care to match the copyright headers with our existing codebase? 

For example, this copyright header in our existing code base would be written like this:
Created by Elizabeth Bartusiak (t-elbart) on 2020-06-12
Copyright (C) Microsoft. All rights reserved.

Things different are:
- Date format
- No year for copyright
- Does not include Project name

Reply:
I'm fine with the current format. Open source has slightly different requirements.